lib/dynamic_time_zone/time_with_zone_patch.rb

```diff
@@ -9,7 +9,14 @@ def localtime(utc_offset = nil)
   alias_method :getlocal, :localtime
 
   def formatted_offset(*args)
-    using_dynamic_time_zone? ? '+0000' : super
+    actual_formatted_offset = super
+    offset_is_greater_than_24_hours = actual_formatted_offset.to_i.abs > 24 * 60 * 60
```

Comment:
This can use method `utc_offset` in `class ActiveSupport::TimeWithZone`

Reply:
Nice :+1:
Changed it.